bot/utils/converters.py

```diff
@@ -71,4 +71,39 @@ async def convert(self, ctx: Context, claim: str) -> Claims:
             return Claims.__members__[claim]
         except KeyError:
             raise ConversionError(f'`{claim}` is not a valid Claim')
-        
\ No newline at end of file
+
+class HonorsConverter(Converter):
+    """Sanitize honors argument input for grades_cog"""
+
+    async def convert(self, ctx: Context, argument: str) -> str:
+        honors = None
+
+        if argument in ('honors', 'hon'):
+            honors = 'honors'
+        elif argument in ('non-honors', 'non-hon', 'regular', 'normal'):
+            honors = 'non-honors'
+        elif argument == 'all':
+            honors = 'all'
+        else:
+            raise ParserError('Are you sure you used the proper format (ex: math-2060 2017 honors)?')
```

Comment:
we should raise a ConversionError here

Reply:
Why should this be a `ConversionError` and not a `BadArgument`? From the docs: https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.BadArgument

> Exception raised when a parsing or conversion failure is encountered on an argument to pass into a command.

Reply:
we use conversion error in the other converters, its for consistency